### PR TITLE
feat: Create NFT DAO Wizard

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**
Screenshot 2023-04-28 at 2 36 06 PM
Screenshot 2023-04-28 at 2 36 21 PM
 
Screenshot 2023-04-28 at 2 36 43 PM
 
Screenshot 2023-04-28 at 2 37 00 PM

### CODE REVIEW

File: src/dex-ui/hooks/hederaTokenService/useCreateNonFungibleToken.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export interface UseCreateNFTParams {
+  name: string;
+  symbol: string;
+  maxSupply: number;
+  treasuryAccountId: string;
+  supplyKey: string;
+}
+
+export function useCreateNonFungibleTokenToken(handleCreateTokenSuccessful: () => void) {
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/hooks/hederaTokenService/useCreateNonFungibleToken.tsx:15]: "Token" is used twice here - not sure if that was intended

File: src/dex-ui/pages/dao/CreateADAO/forms/NFTDAOGovernanceForm.tsx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+          textStyle="b3"
+          message={`If you create a new NFT you will need to manually enter 
+                    the id of that NFT in this form. NFT details can be found using the 
+                    hashscan link provided after a successful NFT creation.`}
+        />
+        <Button key="create-nft" onClick={createNewToken}>
+          Create New NFT
+        </Button>
+      </Flex>
+      <LoadingDialog
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/CreateADAO/forms/NFTDAOGovernanceForm.tsx:223]: You can use conditional states in the `LoadingDialog` props to reduce the number of `LoadingDialog` components from 4 to 2.

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]

File: src/dex-ui/hooks/hederaTokenService/useCreateNonFungibleToken.tsx (hunk 1 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/hooks/hederaTokenService/useCreateNonFungibleToken.tsx]: Done

File: src/dex-ui/pages/dao/CreateADAO/forms/NFTDAOGovernanceForm.tsx (hunk 2 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/CreateADAO/forms/NFTDAOGovernanceForm.tsx:244]: Done

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Pull request merged**